t/01-load.t

```diff
@@ -0,0 +1,15 @@
+#!perl
+
+use strict;
+use warnings;
+use Test::More tests => 6;
+
+BEGIN {
```

Comment:
I assumed the use of `use_ok()` was deprecated a long time ago. See http://blogs.perl.org/users/ovid/2012/04/avoiding-use-ok-in-t00-loadt.html for example. Is it considered ok?